### PR TITLE
[v0.5.0] added metrics to locust

### PR DESCRIPTION
**Pull request by jonaphael**
* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

File: connector/mqtt/locust/src/mqtt_locust/mqtt_client.py [hunk 1]
```diff
@@ -355,6 +391,14 @@ def locust_on_disconnect(self, _client: mqtt.Client, _userdata, result_code: int
                 response_time=0,
                 exception=DisconnectError(Utils.error_message(result_code))
             )
+        # metrics
+        disconnected_time = Utils.seconds_to_milliseconds(time.time()) - self.start_time
+        Utils.fire_locust_failure(
```
Comment (eduardogmisiuk) [inline connector/mqtt/locust/src/mqtt_locust/mqtt_client.py:396]: I don't think it should be counted as a failure.

**Review by eduardogmisiuk — CHANGES_REQUESTED** [review]
Nice! I left only one comment.

**Review by eduardogmisiuk — DISMISSED** [review]

**Review by thiagodpf — APPROVED** [review]
I believe that Eduardo is more inside the logic involved in this code, but in my view, it's ok.

**Comment by mprevide** [comment]
Maybe you can close this PR and open another one to development.  @jonaphael

**Comment by mprevide** [comment]
I change the PR to a feature branch :)
So, now I can merge this PR!

**Review by mprevide — APPROVED** [review]

**Pull request merged**